### ISSUE TITLE
what is the compatiable minecraft server version?

### ISSUE THREAD

**Issue by sebasdt** [AddstarMC/BlockHunt#63]
Hi. I know this plugin is old and probably abandoned but its the only plugin I could find 
with the old hide and seek.

I've tried every paper server version from 1.8.8 through 1.17.1 and none of them lets the plugin work properly.

I'm not able to do any tests since Since in the first I not able to create an arena!
An internal error occurred while attempting to perform this command while
or console: 
![image](https://user-images.githubusercontent.com/67417070/150693275-55d42c8e-f041-4168-ad93-8b2b0591fda9.png)

**Comment by addstar** [comment]
> I've tried every paper server version from 1.8.8 through 1.17.1 and none of them lets the plugin work properly.

Our Blockhunt server is 1.17.1 and working fine, but I'm not sure when we've last tried to create an arena. Maybe the arena creation is broken.

Can you give the exact 1.17.1 server type/build you're trying to run on? And where you got the Blockhunt build from?

**Comment by sebasdt** [comment]
Hi!
Every paper server version I tried was the  latest papermc build and downloaded the plugin jar file from the README of this repo.

One thing too it's totally not clear what version to use with this plugin! The README says 1.12.2 and here you are running it in1.17.1 😐

**Comment by addstar** [comment]
> One thing too it's totally not clear what version to use with this plugin! The README says 1.12.2 and here you are running it in1.17.1 neutral_face

Yeah sorry for the confusion. We forked this from the original creator and maintained it for our own servers because it was discontinued. We'll update the README sometime, but most of that was from the original author.

It definitely works on 1.17.1 but the problem may just be the creation of arenas. I'll test it and if it's broken, I'll update it to work with 1.17/1.18

**Comment by sebasdt** [comment]
Cool cool!
I've got it working on 1.17.1 and already played multiple games with 6 buddies of mine.
The things we've noticed are:
Sadly no use of taunts (makes playing medium maps almost impossible) and ~~hiders missing the ability to defend themselves~~ (~~hiders PVP broken?~~ Just needed to set "hidersCanHurtSeekers" to true in plugins/blockhunt/arenas.yml)

> I'll update it to work with 1.17/1.18

If possible could u do support for both versions? with a preference for 1.17.

also, I ploughed through the docs and saw there was soft support for the multiverse plugin, is that right?

Edit:
hiders can attack seekers now. the variable hidersCanHurtSeekers needs to be set to true.
